src/invoices/InvoiceDetails/ApproveConfirmationModal/ApproveConfirmationModal.js

```diff
@@ -18,11 +18,14 @@ const ApproveConfirmationModal = ({
   onCancel,
   onConfirm,
   invoice,
+  heading,
```

Comment:
`heading` may be misunderstood as a complete string or node to display in DOM, smth like `headingLabelId` may be a more suitable term